app/main/views/platform_admin.py

```diff
@@ -771,3 +781,76 @@ def _get_user_row(r):
     row.append(r["password_changed_at"])
     row.append(r["state"])
     return row
+
+
+@main.route(
+    "/platform-admin/load-test",
+    methods=["POST", "GET"],
+)
+@user_is_platform_admin
+def load_test():
+    """
+    The load test assumes that a service called 'Test service' exists.  It will make
+    the platform admin a member of this service if the platform is not already. All
+    messagese will be sent in this service.
+    """
+    # SIMULATED_SMS_NUMBERS = ("+14254147755", "+14254147167")
+    service = _find_load_test_service()
+    _prepare_load_test_service(service)
+    example_template = _find_example_template(service)
+
+    for _ in range(0, 3):
+        session["recipient"] = "+14254147755"
+        session["placeholders"] = {
+            "day of week": "Monday",
+            "color": "blue",
+            "phone number": "+14254147755",
+        }
+        _send_notification(service["id"], example_template["id"])
+    for _ in range(0, 3):
+        session["recipient"] = "+14254147167"
+        session["placeholders"] = {
+            "day of week": "Monday",
+            "color": "blue",
+            "phone number": "+14254147167",
+        }
+        _send_notification(service["id"], example_template["id"])
```

Comment:
I'd add a couple comments to each `for` block just to note which is the success and which is the failure - or, we can swap the numbers out for the config variables and/or constants/enums that are clearly labeled too.  Food for future thought!